CONTRIBUTORS.md

```diff
@@ -32,6 +32,48 @@ This project and everyone participating in it is governed by this [Code of Condu
 - For consensus breaking changes, branch off of the `next` branch.
 - For hotfixes, branch off of `master`.
 
+### GitHub Tags
+We use tags on PRs and issues to categorize them. You should try to select a tag from each category below if possible. 
+
+**Timeline- and consensus- based tags:**
+- Next non-breaking change => `ship now`
```

Comment:
can rename to "next: non-breaking"

Reply:
I don't think we should use `next` in multiple tags, because it also refers to a branch (which is the next consensus version). `ship now` works better I think.

Reply:
- agree on not using next name
- however, ship now doesn't communicate that change is consensus-compatible

Reply:
I think by default changes tend to be consensus-compatible, so I will leave it as `ship now` (unless I get a better suggestion)